.github/workflows/integration-test.yml

```diff
@@ -97,5 +68,5 @@ jobs:
         if: always()
         uses: actions/upload-artifact@v3
         with:
-          name: logs-integration-tests-scheduling-gates
+          name: logs-integration-tests-${{ inputs.helm-install }}-${{ inputs.scheduling_gates }}
```

Comment:
can we have an other step in which we setup an env var for the create reports step? this (true true ) is not really user friendly

Reply:
sure, we can add prefix there